### PR TITLE
PLANNER-2190: Document Coach Shuttle Gathering example

### PR DESCRIPTION
**Pull request by Christopher-Chianelli**
### JIRA

https://issues.redhat.com/browse/PLANNER-2190

### Checklist
- [ ] Documentation updated if applicable.
- [ ] Upgrade recipe provided if applicable.

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* for a <b>pull request build</b> please add comment: <b>Jenkins retest this</b>
* for a <b>full downstream build</b> please add comment: <b>Jenkins run fdb</b>
* for a <b>compile downstream build</b> please add comment: <b>Jenkins run cdb</b>
* for a <b>full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>
* for an <b>upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

File: optaplanner-docs/src/main/asciidoc/UseCasesAndExamples/CoachShuttleGathering/CoachShuttleGathering-section.adoc [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+distance for coaches that go directly to the hub
+
+[[coachShuttleGatheringProblemSize]]
+== Problem size
+
+[source,options="nowrap"]
+----
+CoachShuttleGathering example has 20 road locations, 3 coaches, 6 shuttles and 12 bus stops with a search space of 10^23.
+CoachShuttleGathering public_preselection_1 has 193 road locations, 6 coaches, 333 shuttles and 144 bus stops with a search space of 10^1311.
+CoachShuttleGathering public_preselection_2 has 193 road locations, 6 coaches, 333 shuttles and 144 bus stops with a search space of 10^1311.
```
Comment (ge0ffrey) [inline optaplanner-docs/src/main/asciidoc/UseCasesAndExamples/CoachShuttleGathering/CoachShuttleGathering-section.adoc:51]: Would you mind adding spaces after `example` to align the `has` on all 3 lines.
Also add an extra space before `20`, 2 spaces before `6` and 1 space before `12` so all numbers are right aligned like in the other docs?

**Review by ge0ffrey — APPROVED** [review]
Good work, good to merge once numbers are aligned in problem size section.

**Comment by triceo** [comment]
@Christopher-Chianelli Mind finishing this? Let's not leave it to rot, it is useful.

**Comment by Christopher-Chianelli** [comment]
@ge0ffrey @triceo right aligned name and numbers in the example.

**Pull request merged**